Mathlib/CategoryTheory/Products/Basic.lean

```diff
@@ -352,4 +353,25 @@ def functorProdFunctorEquiv : (A ⥤ B) × (A ⥤ C) ≌ A ⥤ B × C :=
     counitIso := functorProdFunctorEquivCounitIso A B C, }
 #align category_theory.functor_prod_functor_equiv CategoryTheory.functorProdFunctorEquiv
 
+section Opposite
+
+open Opposite
+
+/-- The equivalence between the opposite of a product and the product of the opposites. -/
+@[simps]
+def prodOp : (C × D)ᵒᵖ ≌ Cᵒᵖ × Dᵒᵖ where
```

Comment:
I would suggest using the name `prodOpEquivalence` instead.

Reply:
Right, the definition above uses `Equiv` so maybe it is best to pick `prodOpEquiv`.